yahtzee.py

```diff
@@ -197,28 +246,44 @@ def __init__(self):
             
         print('You rolled: ', player1.current_throw)
     
-    
+ 
     for i, category in enumerate(scorecard.categories):
         category.is_possible(player1.current_throw)
         category.score(player1.current_throw)
```

Comment:
Here I see why you defined all the classes separately and I think this is an awesome feature. This way you can easily access the is_possible and score, etc